common/dev/tmp75.c

```diff
@@ -0,0 +1,41 @@
+#include "tmp75.h"
+
+#include <stdio.h>
+
+#include "sensor.h"
+#include "hal_i2c.h"
+
+#define RETRY 5
+
+bool tmp75_read(uint8_t sensor_num, float *reading)
```

Comment:
Check reading != NULL

Reply:
The optimization code is based on v2022.07.01. After that, the main branch has been phased in some
commits related to sensor monitoring. Those changes will be phased in main_v2 in the other PRs. This
problem will be solved in those PRs.
Please check #223  .